content/posts.md

```diff
@@ -0,0 +1,7 @@
++++
+title = 'Posts'
+date = 2023-10-21T22:18:56+05:30
+draft = true
++++
+
```

Comment:
Here, we should be redirecting the user to my blog page instead of loading on the same content.